config/attribute-map.yml

```diff
@@ -0,0 +1,3 @@
+"urn:mace:dir:attribute-def:uid": "uid"
```

Comment:
What if we want other values, like first name, last name, etc?

Reply:
See: https://www.purdue.edu/securepurdue/identity-access/shibboleth-service.php
Attribute: displayName